UserDetailsClient/UserDetailsClient.Core/MainPage.xaml.cs

```diff
@@ -73,7 +56,7 @@ private IUser GetUserByPolicy(IEnumerable<IUser> users, string policy)
         {
             foreach (var user in users)
             {
-                string userIdentifier = Base64UrlDecode(user.Identifier.Split('.')[0]);
```

Comment:
Why remove Base64UrlDecode? is it already done by the library? Wasn't it the case before?

Reply:
because of the Base64UrlDecode, the policy is getting decoded, so it never identifies a user by policy. Removing this allows the `GetUserByPolicy` to determine the user based on the policy.